package.json

```diff
@@ -3,6 +3,10 @@
   "version": "0.1.0",
   "description": "Momento read-aside cache client for sequelize",
   "main": "dist/index.js",
+  "types": "dist/index.d.ts",
+  "files": [
+    "dist"
+  ],
```

Comment:
this i believe fixes recognizing the files.. it was being recognized under `dist` before